### DISCUSSION TITLE
In release 2.0.0 change wikidocs folders structure

### DISCUSSION THREAD

**Discussion by ffiesta** [Zavy86/WikiDocs#227] [Ideas]
Hi, 

I think to better secure wikidocs and prevent some "web hacks sniffing", wikidocs need to change based structure of folders, to stay more like filegator struture.

![image](https://github.com/user-attachments/assets/0d268545-77f8-44f3-8990-3308281a4ed1)

**Comment by Zavy86** [comment]
it's a good idea
